llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -519,13 +591,20 @@ class InstrProfSymtab {
   /// will be represented using the same StringRef value.
   inline StringRef getFuncNameOrExternalSymbol(uint64_t FuncMD5Hash);
 
+  /// Just like getFuncName, except that it will return a non-empty StringRef
```

Comment:
Cleanup

Reply:
Cleaned this up.

I just realized `getFuncNameOrExternalSymbol` and  `getFuncName` (line 587) needs a rename. These two functions are used in many places (llvm-profdata, text profile writer, coverage mapping, and unit test). I'm going to send out an NFC for the rename.